### ISSUE TITLE
ProfileView will not build in juliapro 1.0.3.1 on OSX 10.14.4: segfaults

### ISSUE THREAD

**Issue by mgalenb** [timholy/ProfileView.jl#110]
When i try to add ProfileView to a fresh install of juliapro 1.0.3.1 i get the following error when building Cairo:
```
Building Cairo ───→ `~/.juliapro/JuliaPro_v1.0.3.1/packages/Cairo/CXPG1/deps/build.log`
┌ Error: Error building `Cairo`, showing the last 100 of log:
│ You can install Python packages with
│   pip3 install <package>
│ They will install into the site-package directory
│   /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/lib/python3.7/site-packages
│
│ See: https://docs.brew.sh/Homebrew-and-Python
│ ==> Summary
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/python/3.7.3: 3,863 files, 59.9MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/python/3.7.3
│ ==> Installing libffi from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/libffi-3.2.1.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/89b4ef328be250153e7a2b39982ac446e0b3bbfe7f3e2708912ba1976862c302--libffi-3.2.1.mojave.bottle.tar.gz
│ ==> Pouring libffi-3.2.1.mojave.bottle.tar.gz
│ ==> Caveats
│ libffi is keg-only, which means it was not symlinked into /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr,
│ because some formulae require a newer version of libffi.
│
│ For compilers to find libffi you may need to set:
│   export LDFLAGS="-L/Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/opt/libffi/lib"
│
│ ==> Summary
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/libffi/3.2.1: 16 files, 297KB
│ ==> Installing glib from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/glib-2.60.0_1.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/1c92dc54c48ba5a2d12d9498a9115546dfcaf257650a7baf6ec67911f60e8c2e--glib-2.60.0_1.mojave.bottle.tar.gz
│ ==> Pouring glib-2.60.0_1.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/glib/2.60.0_1: 432 files, 15.3MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/glib/2.60.0_1
│ ==> Installing libpng from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/libpng-1.6.36.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/b401f5b07a537975da7de448c4bbdaf0c2112d5dc7d8e0955e0d4fb938909010--libpng-1.6.36.mojave.bottle.tar.gz
│ ==> Pouring libpng-1.6.36.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/libpng/1.6.36: 27 files, 1.2MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/libpng/1.6.36
│ ==> Installing freetype from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/freetype-2.10.0.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/b11d24b3f88092e926183279c927c8f3b1ed6c1044a2828a6c7474513fdd567b--freetype-2.10.0.mojave.bottle.tar.gz
│ ==> Pouring freetype-2.10.0.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/freetype/2.10.0: 61 files, 2.3MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/freetype/2.10.0
│ ==> Installing fontconfig from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/fontconfig-2.13.1.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/b7ad94d49bcdca22738f50b570efc07c7a20f98a469b3c4b32198b7318b8af5a--fontconfig-2.13.1.mojave.bottle.tar.gz
│ ==> Pouring fontconfig-2.13.1.mojave.bottle.tar.gz
│ ==> Regenerating font cache, this may take a while
│ ==> /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/fontconfig/2.13.1/bin/fc-cache -frv
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/fontconfig/2.13.1: 531 files, 3.3MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/fontconfig/2.13.1
│ ==> Installing pixman from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/pixman-0.38.0.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/0c26d4c3daf0b26cd91366caf1df3eca0bad5611e9742a0f8b409d6b80fc6a53--pixman-0.38.0.mojave.bottle.tar.gz
│ ==> Pouring pixman-0.38.0.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/pixman/0.38.0: 13 files, 1.3MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/pixman/0.38.0
│ ==> Installing cairo from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/cairo-1.16.0.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/fafc9fd181a939883ef055697994edea57d091a35299109bf774fc91092fbae7--cairo-1.16.0.mojave.bottle.tar.gz
│ ==> Pouring cairo-1.16.0.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/cairo/1.16.0: 117 files, 5.5MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/cairo/1.16.0
│ ==> Installing icu4c from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/icu4c-63.1.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/88108185da55bf1af475d5e191e97f633514bd24c1c70bb89029af6cf5cc5185--icu4c-63.1.mojave.bottle.tar.gz
│ ==> Pouring icu4c-63.1.mojave.bottle.tar.gz
│ ==> Caveats
│ icu4c is keg-only, which means it was not symlinked into /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr,
│ because macOS provides libicucore.dylib (but nothing else).
│
│ If you need to have icu4c first in your PATH run:
│   echo 'export PATH="/Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/opt/icu4c/bin:$PATH"' >> ~/.bash_profile
│   echo 'export PATH="/Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/opt/icu4c/sbin:$PATH"' >> ~/.bash_profile
│
│ For compilers to find icu4c you may need to set:
│   export LDFLAGS="-L/Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/opt/icu4c/lib"
│   export CPPFLAGS="-I/Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/opt/icu4c/include"
│
│ ==> Summary
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/icu4c/63.1: 254 files, 68.4MB
│ ==> Installing fribidi from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/fribidi-1.0.5.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/b966c6f589c81500014b64f8e15ff29e9b70f3e7ee8e0b0b5851f700c2ad0fa4--fribidi-1.0.5.mojave.bottle.tar.gz
│ ==> Pouring fribidi-1.0.5.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/fribidi/1.0.5: 66 files, 581.4KB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/fribidi/1.0.5
│ ==> Installing harfbuzz from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/harfbuzz-2.4.0.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/7ed01d99e17d4a5b7e46172b0d90b69b2252abe6619e962aab6867ac0eec0b1f--harfbuzz-2.4.0.mojave.bottle.tar.gz
│ ==> Pouring harfbuzz-2.4.0.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/harfbuzz/2.4.0: 145 files, 9.5MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/harfbuzz/2.4.0
│ ==> Installing pango from staticfloat/juliatranslated
│ ==> Downloading https://homebrew.bintray.com/bottles/pango-1.42.4_1.mojave.bottle.tar.gz
│ Already downloaded: /Users/galen/Library/Caches/Homebrew.jl/downloads/f7d576fd30d48fab15ea38e97c79854a517864b3a64199be1550eb79e1143cf3--pango-1.42.4_1.mojave.bottle.tar.gz
│ ==> Pouring pango-1.42.4_1.mojave.bottle.tar.gz
│ 🍺  /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/pango/1.42.4_1: 106 files, 4.4MB
│ Warning: Already linked: /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Homebrew/s09IX/deps/usr/Cellar/pango/1.42.4_1
│
│ signal (11): Segmentation fault: 11
│ in expression starting at /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Cairo/CXPG1/deps/build.jl:165
│ _platform_strcmp at /usr/lib/system/libsystem_platform.dylib (unknown line)
│ Allocations: 20232142 (Pool: 20228886; Big: 3256); GC: 45
│
│ Full log at /Users/galen/.juliapro/JuliaPro_v1.0.3.1/packages/Cairo/CXPG1/deps/build.log
└ @ Pkg.Operations /Users/vagrant/worker/juliapro-release-osx1011-0_6/build/tmp_julia/Julia-1.0.app/Contents/Resources/julia/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1136
  Building Gtk ─────→ `~/.juliapro/JuliaPro_v1.0.3.1/packages/Gtk/ADDrn/deps/build.log`
  Building WebIO ───→ `~/.juliapro/JuliaPro_v1.0.3.1/packages/WebIO/Rk8wc/deps/build.log`
```

**Comment by timholy** [comment]
Technically this is not a ProfileView bug, it's a Cairo/Homebrew bug: https://github.com/JuliaGraphics/Cairo.jl/issues/271. I'll close this.

**Comment by timholy** [comment]
Should be fixed if you use Julia 1.3 and up-to-date packages
